README.md

```diff
@@ -506,16 +507,18 @@ message FeeStructure {
   TradeFees maker = 1;
   TradeFees taker = 2;
   int32 clear_write_notional_bps = 3;
-  int32 clear_redeemed_notional_bps = 4;
+  int32 clear_redeem_notional_bps = 4;
```

Comment:
For better or worse, this is not the case, and we aren't going to make a breaking API change to fix right this second, but let's get something in the backlog.